src/styles/components/_definition.scss

```diff
@@ -1,35 +1,52 @@
 $breakpoint-tablet: 820px;
 
 .cmp-definition {
-  min-height: 20rem;
+  position: relative;
+  line-height: 1.75rem;
 
   &__heading {
     position: relative;
     font-family: Viga, sans-serif;
     font-weight: 400;
     font-size: map.get($font-sizes, "md");
+
+    /* Circle around "definition" */
+    &::after {
```

Comment:
Nice use of ::after!

Reply:
I think it was Monet that recommended I swap this to an ::after element. It makes for weirder-looking css imo, but much cleaner jsx.